src/hooks/use-archive-site.ts

```diff
@@ -84,7 +84,7 @@ export function useArchiveSite() {
 						__(
 							'The site exceeds the maximum size of %dMB. Please remove some files and try again.'
 						),
-						Math.floor( LIMIT_ARCHIVE_SIZE / 1024 / 1024 )
+						Math.floor( SIZE_LIMIT_MB )
```

Comment:
Should we remove the Math.floor ?
```suggestion
						SIZE_LIMIT_MB
```

Reply:
Done: https://github.com/Automattic/studio/pull/446/commits/daa91834e8fb78f0eeb6048d63306d7953a5e6e2